02-analytics/technical.md

```diff
@@ -243,7 +196,7 @@ Use `/track` endpoint. Docs: https://docs.rudderstack.com/rudderstack-api/api-sp
 
 | Data        | Name             | Comments                     |
 |-------------|------------------|------------------------------|
-| Event ID    | event            | Hardcoded: `video_toggle`    |
+| Event ID    | event            | Hardcoded: `audioonly_clicked`    |
 | Video state | videoStateTarget | Video state *after* tap      |
```

Comment:
What is video state? 
"video" | "audio_only" ?

Reply:
```suggestion
| Audio only enabled | audioOnly | Audio only state *after* tap. true or false      |
```